examples/twittermap/noah/src/test/scala/edu/uci/ics/cloudberry/noah/TwitterJSONTagToADMTest.scala

```diff
@@ -9,9 +9,9 @@ import edu.uci.ics.cloudberry.util.Profile._
 class TwitterJSONTagToADMTest extends FlatSpec with Matchers {
   "USGeoGnosis" should "tag the us json file" in {
     val sLoadGeo = new Date()
-    val shapeMap = Map.apply(StateLevel -> new File("web/public/data/state.json"),
-      CountyLevel -> new File("web/public/data/county.json"),
-      CityLevel -> new File("web/public/data/city.json"))
+    val shapeMap = Map.apply(StateLevel -> new File("/root/cloudberry/examples/twittermap/web/public/data/state.json"),
+      CountyLevel -> new File("/root/cloudberry/examples/twittermap/web/public/data/county.json"),
+      CityLevel -> new File("/root/cloudberry/examples/twittermap/web/public/data/city.json"))
```

Comment:
Why is this change necessary?

Reply:
The Jackson JSON Parser need to load the Class Object to match the JSON object while parsing JSON string, so I created a new MyPlace.java. Since we don't use Twitter JSON parser anymore, the Place.java is also unnecessary.
"/root/cloudberry/...." is unnecessary, just used in my test environment, sorry to forgot to delete them.